tests/near-bindgen/dist/index.js

```diff
@@ -0,0 +1,514 @@
HlwZXMgPSAodHlwZXM6IHN0cmluZ1tdKTogU2ltcGxlRmllbGRbXSA9PlxuICAgICAgZmllbGRzLmZpbHRlcihmaWVsZCA9PiB0eXBlcy5pbmRleE9mKHRoaXMudHlwZU5hbWUoZmllbGQudHlwZSkpICE9IC0xKTtcblxuICAgIHRoaXMuZ2VuZXJhdGVCYXNpY1NldHRlckhhbmRsZXJzKFxuICAgICAgdmFsdWVQcmVmaXgsXG4gICAgICBcIkludGVnZXJcIixcbiAgICAgIFwiaTY0XCIsXG4gICAgICBmaWVsZHNXaXRoVHlwZXMoW1wiaTMyXCIsIFwidTMyXCJdKVxuICAgICk7XG4gICAgdGhpcy5nZW5lcmF0ZUJhc2ljU2V0dGVySGFuZGxlcnMoXG4gICAgICB2YWx1ZVByZWZpeCxcbiAgICAgIFwiU3RyaW5nXCIsXG4gICAgICBcInN0cmluZ1wiLFxuICAgICAgZmllbGRzV2l0aFR5cGVzKFtcInN0cmluZ1wiLCBcImk2NFwiLCBcInU2NFwiLCBcIlVpbnQ4QXJyYXlcIiwgXCJ1MTI4XCJdKVxuICAgICk7XG4gICAgdGhpcy5nZW5lcmF0ZUJhc2ljU2V0dGVySGFuZGxlcnMoXG4gICAgICB2YWx1ZVByZWZpeCxcbiAgICAgIFwiQm9vbGVhblwiLFxuICAgICAgXCJib29sXCIsXG4gICAgICBmaWVsZHNXaXRoVHlwZXMoW1wiYm9vbFwiXSlcbiAgICApO1xuXG4gICAgdmFyIG5vbkJhc2ljRmllbGRzID0gZmllbGRzLmZpbHRlcihmaWVsZCA9PiBpc1JlZmVyZW5jZShmaWVsZC50eXBlKSk7XG5cbiAgICB0aGlzLnNiLnB1c2goXCJzZXROdWxsKG5hbWU6IHN0cmluZyk6IHZvaWQge1wiKTtcbiAgICBub25CYXNpY0ZpZWxkcy5mb3JFYWNoKGZpZWxkID0+IHtcbiAgICAgIHRoaXMuc2IucHVzaChgaWYgKG5hbWUgPT0gXCIke2ZpZWxkLm5hbWV9XCIpIHtcbiAgICAgICAgICAke3ZhbHVlUHJlZml4fSR7ZmllbGQubmFtZX0gPSA8JHt0aGlzLnR5cGVOYW1lKGZpZWxkLnR5cGUpfT5udWxsO1xuICAgICAgICAgIHJldHVybjtcbiAgICAgICAgfWApO1xuICAgIH0pO1xuICAgIHRoaXMuc2IucHVzaChgXG4gICAgICAgIHN1cGVyLnNldE51bGwobmFtZSk7XG4gICAgICB9YCk7XG5cbiAgICB0aGlzLnNiLnB1c2goYFxuICAgICAgICBwdXNoT2JqZWN0KG5hbWU6IHN0cmluZyk6IGJvb2wge2ApO1xuICAgIHRoaXMuc2IucHVzaChgaWYgKCF0aGlzLmhhbmRsZWRSb290KSB7XG4gICAgICAgIGFzc2VydChuYW1lID09IG51bGwgfHwgbmFtZS5sZW5ndGggPT0gMCk7XG4gICAgICAgIHRoaXMuaGFuZGxlZFJvb3QgPSB0cnVlO1xuICAgICAgICByZXR1cm4gdHJ1ZTtcbiAgICAgIH0gZWxzZSB7XG4gICAgICAgIGFzc2VydChuYW1lICE9IG51bGwgfHwgbmFtZS5sZW5ndGggIT0gMCk7XG4gICAgICB9YCk7XG4gICAgdGhpcy5nZW5lcmF0ZVB1c2hIYW5kbGVyKFxuICAgICAgdmFsdWVQcmVmaXgsXG4gICAgICBub25CYXNpY0ZpZWxkcy5maWx0ZXIoZmllbGQgPT4gIXRoaXMuaXNBcnJheVR5cGUoZmllbGQudHlwZSkpXG4gICAgKTtcbiAgICB0aGlzLnNiLnB1c2goYFxuICAgICAgICAgIHJldHVybiBzdXBlci5wdXNoT2JqZWN0KG5hbWUpO1xuICAgICAgICB9YCk7XG4gICAgdGhpcy5zYi5wdXNoKGBcbiAgICAgICAgcHVzaEFycmF5KG5hbWU6IHN0cmluZyk6IGJvb2wge2ApO1xuICAgIHRoaXMuZ2VuZXJhdGVQdXNoSGFuZGxlcihcbiAgICAgIHZhbHVlUHJlZml4LFxuICAgICAgbm9uQmFzaWNGaWVsZHMuZmlsdGVyKGZpZWxkID0+IHRoaXMuaXNBcnJheVR5cGUoZmllbGQudHlwZSkpXG4gICAgKTtcbiAgICB0aGlzLnNiLnB1c2goYFxuICAgICAgICAgIHJldHVybiBzdXBlci5wdXNoQXJyYXkobmFtZSk7XG4gICAgICAgIH1gKTtcbiAgfVxuXG4gIHByaXZhdGUgZ2VuZXJhdGVCYXNpY1NldHRlckhhbmRsZXJzKFxuICAgIHZhbHVlUHJlZml4OiBzdHJpbmcsXG4gICAgc2V0dGVyVHlwZTogc3RyaW5nLFxuICAgIHNldHRlclZhbHVlVHlwZTogc3RyaW5nLFxuICAgIG1hdGNoaW5nRmllbGRzOiBTaW1wbGVGaWVsZFtdXG4gICkge1xuICAgIGlmIChtYXRjaGluZ0ZpZWxkcy5sZW5ndGggPiAwKSB7XG4gICAgICB0aGlzLnNiLnB1c2goXG4gICAgICAgIGBzZXQke3NldHRlclR5cGV9KG5hbWU6IHN0cmluZywgdmFsdWU6ICR7c2V0dGVyVmFsdWVUeXBlfSk6IHZvaWQge2BcbiAgICAgICk7XG4gICAgICBtYXRjaGluZ0ZpZWxkcy5mb3JFYWNoKGZpZWxkID0+IHtcbiAgICAgICAgLy8gdHNsaW50OmRpc2FibGUtbmV4dC1saW5lOiBhcy12YXJpYWJsZXNcbiAgICAgICAgbGV0IGZpZWxkVHlwZU5hbWUgPSB0aGlzLnR5cGVOYW1lKGZpZWxkLnR5cGUpO1xuICAgICAgICBpZiAoc2V0d
```

Comment:
What is this

Reply:
Ah that's the inline source mapping, which is useful for debugging.  Perhaps I should separate the tsconfig so that a production build doesn't have it.

Reply:
@willemneal I'd prefer we have source mappings in the production build, but in a separate file (so browser doesn't have to load them unless needed).